packages/server/src/tests/header/headerSelection.ts

```diff
@@ -27,35 +27,45 @@ const nonSupportersTestNonUK: HeaderTest = {
     ],
 };
 
-const nonSupportersTestUS: HeaderTest = {
-    name: 'RemoteRrHeaderLinksTest__USEOY',
-    audience: 'AllNonSupporters',
-    variants: [
-        {
-            name: 'remote',
-            modulePathBuilder: usEOYHeader.endpointPathBuilder,
-            content: {
-                heading: 'Support the Guardian',
-                subheading: 'Make a year-end gift',
-                primaryCta: {
-                    url: 'https://support.theguardian.com/contribute',
-                    text: 'Contribute',
-                },
-                secondaryCta: {
-                    url: 'https://support.theguardian.com/subscribe',
-                    text: 'Subscribe',
+const nonSupportersTestUS = (): HeaderTest => {
+    const givingTuesdayStart = new Date('2021-11-29T17:00:00'); //remove "Subscribe" Monday 12:00 PM EST
+    const givingTuesdayEnd = new Date('2021-11-01T09:00:00'); //re-add "Subscribe" on Wednesday morning GMT
+    const currentDateTime = new Date();
+    const shouldShowSubscribeButton =
+        givingTuesdayStart <= currentDateTime && currentDateTime <= givingTuesdayEnd;
+
```

Comment:
`shouldShowSubscribeButton` should be `false` during Giving Tuesday, however with this logic I think it will be `true`. 

For example, if the `currentDateTime` is `2021-11-30`, then `givingTuesdayStart <= currentDateTime` will be `true` and `currentDateTime <= givingTuesdayEnd` will also be `true` - so `shouldShowSubscribeButton` would be `true` on `2021-11-30`, which is not what we want?

I believe the below will set `shouldShowSubscribeButton` to `true` when `currentDateTime` is before the `givingTuesdayStart` OR `currentDateTime` is after the `givingTuesdayEnd`, and it'll return false when it's between those dates...

```
const shouldShowSubscribeButton = currentDateTime <= givingTuesdayStart || currentDateTime >= givingTuesdayEnd;
```

Reply:
yeah it should be!

Reply:
Thanks @GHaberis, @marialani and I are just working on a new PR to fix this.

Reply:
Yeah we were initially using the date-fns library but the 0 based indexing for months was a bit confusing so this was updated to use the Date object but some of the logic had been muddled in translation. This has since been updated on [this PR](https://github.com/guardian/support-dotcom-components/pull/585)